app/models/event.rb

```diff
@@ -38,7 +38,7 @@ class Event < ApplicationRecord
   validate :abstract_limit
   validate :before_end_of_conference, on: :create
   validates :title, presence: true
-  validates :abstract, presence: true
+  validates :abstract, presence: true, if: 'event_type.minimum_abstract_length > 0'
```

Comment:
what about validating that abstract lenght is at least the `minimum_abstract_length` instead of doing this only for the empty case?

Reply:
@Ana06 As far as i understand you are saying that i should validate for abstract_length to at least `minimum_abstract_length` which is already done in https://github.com/openSUSE/osem/blob/master/app/models/event.rb#L38
Problem here is that when `minimum_abstract_length` is set to 0 words and `abstract_length` is also 0, code is validating for the presence of the abstract but our abstract is `nil` so we should not validate the presence of abstract when `minimum_abstract_length` is 0 and `abstract_length` is also 0 which is what i am trying to do here.
Please let me know if i misunderstood something!

Reply:
@namangupta01 I'd remove this validation, as we don't really want that abstract is present. We want that abstract is longer than the minimum length and longer than the maximum and that's all. Instead I'll ensure that this case is covered in [`abstract_limit`](https://github.com/openSUSE/osem/blob/master/app/models/event.rb#L284)

Reply:
BTW, passing string to be evaluated in `:if` and `:unless` conditional in validations has been removed in Rails 5.2, so we shouldn't use to make updating easier. :wink:

Reply:
BTW, passing string to be evaluated in `:if` and `:unless` conditional in validations has been removed in Rails 5.2, so we shouldn't use to make updating easier. :wink: